src/main/java/org/isf/menu/gui/MainMenu.java

```diff
@@ -67,23 +64,20 @@
 public class MainMenu extends JFrame implements ActionListener, Login.LoginListener, SubMenu.CommandListener {
 
 	private static final long serialVersionUID = 7620582079916035164L;
-	public static final String ADMIN_STR = "admin";
-	private boolean flag_Xmpp;
-	private boolean flag_Sms;
+	private boolean flag_Xmpp = false;
+	private boolean flag_Sms = false;
 
-	private static final Logger LOGGER = LoggerFactory.getLogger(MainMenu.class);
+	private final Logger logger = LoggerFactory.getLogger(MainMenu.class);
```

Comment:
correct version is with initial values and static LOGGER

Reply:
I restored the current informatici version